Updated Migrating regions cloud article to use new backup feature and import/export features on cloud and deploy. (#6394)
umbraco/UmbracoDocs: umbraco-cloud/getting-started/migrate-between-regions.md

## Prerequisites

* Admin access and deployment rights on the project that is to be migrated.
* A clone of both East US and West EU projects.
* A local setup that can run an Umbraco instance. Learn more about this in the [Requirements](https://docs.umbraco.com/umbraco-cms/fundamentals/setup/requirements) article.

{% hint style="info" %}

> If you want to migrate a Umbraco 8 project, you will need to upgrade to the latest supported [Long-Term-Supported (LTS)](https://umbraco.com/products/knowledge-center/long-term-support-and-end-of-life/) version of Umbraco CMS.

Reviewer: ```suggestion
If you want to migrate an Umbraco 8 project, you will need to upgrade to the latest supported [Long-Term-Supported (LTS)](https://umbraco.com/products/knowledge-center/long-term-support-and-end-of-life/) version of Umbraco CMS.
```

Make sure that your projects are [prepared for migration](migrate-between-regions.md#prepare-your-projects) before continuing the process.
{% endhint %}

> ### Step 1: Creating and restore database backup

Reviewer: ```suggestion
### Step 1: Create and Restore Database Backup
```

1. Go to **Configuration** > **Backups** on the **West EU** Cloud project.
2. Create a **backup** of the projects database.
3. Download the backup to your local machine.
4. Go to the **East US** project.
5. Go to **Configuration** > **Backups**.
6. Upload the **database backup** that you created in the previous step to the project.
7. Restore the **backup** to your environment
   -  **Optional** Create a backup of the environment before restoring the backup.

> 8. Run a **Export Schema** and then **Update Umbraco Schema** from the **Deploy Dashboard** in the settings section of the **East US** project.

Reviewer: ```suggestion
8. Run **Export Schema** from the **Deploy Dashboard** in the **Settings** section of the **East US** project.
9. Run **Update Umbraco Schema** from the **Deploy Dashboard** in the **Settings** section of the **East US** project.
```

Once you have restored the database to your environment, go to the backoffice of the project you are migrating to. In the backoffice, you should now see your content in the Content section, Document Types, and Data Types in the Settings section.

Taking a closer look at the templates, stylesheets, scripts and media, you will notice that it is not there. In the next step we will migrate those over to our new project

### Step 2: Migrate Files

In this step, we will migrate our files and media items from our project in the EU region. 

1. Clone down both the projects to your local machine.
2. Run the local East US project and restore the content.
3. Open both the project folders for **West EU** and **East US**.
4. Move the **view** files located in the view folder from **West EU** to the view folder in the **East US** project.
   - When prompted replace the existing files. 
5. Move the **CSS** and **Script** files located in the **wwwroot** folder from the **West EU** folder to the **wwwroot** folder in the **East US** project.

>    1. **Optional:** Move files from **App_Plugins** if you have extended the Umbraco Backoffice

Reviewer: ```suggestion
   - **Optional:** Move files from **App_Plugins** if you have extended the Umbraco Backoffice
```

6. Move custom code (Models, Controllers and other relevant code) from the **West EU** to the **East US** project
7. Run the **East US** project locally.

Once you have started the project, it should show your content as it was on the **West EU** project. The only thing missing is the media items, as they have not been migrated yet. Before we can migrated our media items, we need to push the migrated files to the Cloud project.

> ### Step 3: Pushing migrated project to Cloud

Reviewer: ```suggestion
### Step 3: Push the Migrated Project to Cloud
```

> In the following steps we will push the migrated local **East US** project back up to the project on Cloud.

Reviewer: ```suggestion
In the following steps, we will push the migrated local **East US** project back up to the project on Cloud.
```

1. Follow the [Deploying Changes](https://docs.umbraco.com/umbraco-cloud/deployments/local-to-cloud) article to push the Views, CSS and JavaScript files to the Cloud environment.
2. Follow the [Transferring Content, Media, Members, and Forms](https://docs.umbraco.com/umbraco-cloud/deployments/content-transfer#media-items)article to transfer the media items to the cloud project.

> Verify that all schemas, files and content have been successfully deployed to your new _US project_ after the transfer.

Reviewer: ```suggestion
Verify that all schemas, files, and content have been successfully deployed to your new _US project_ after the transfer.
```

### Step 4: Migrate Media Items

> In the following step, we will migrate the media items from the **West EU** blob storage container to the **East US** blob storage container.

Reviewer: ```suggestion
In the following steps, we will migrate the media items from the **West EU** blob storage container to the **East US** blob storage container.
```

Make sure that your projects are [prepared for migration](migrate-between-regions.md#prepare-your-projects) before continuing the process.
{% endhint %}

### Step 1: Creating and restore database backup

> {% tab title="Umbraco 10+" %}

Reviewer: Since the below steps are only for Umbraco 10+, you might want to consider removing the tab markdown.  Also, the steps doesn't appear correctly in  the preview mode but it maybe cz the {% endtabs %} is missing.

Author: I just forgot to remove this 🙈

This article is only for v10+ :)

1. Go to **Configuration** > **Backups** on the **West EU** Cloud project.
2. Create a **backup** of the projects database.
3. Download the backup to your local machine.
4. Go to the **East US** project.
5. Go to **Configuration** > **Backups**.
6. Upload the **database backup** that you created in the previous step to the project.
7. Restore the **backup** to your environment
   -  **Optional** Create a backup of the environment before restoring the backup.
8. Run a **Export Schema** and then **Update Umbraco Schema** from the **Deploy Dashboard** in the settings section of the **East US** project.

> Once you have restored the database to your environment, go to the backoffice of the project you are migrating to. In the backoffice, you should now see your content in the content section, and Document Types and Data Types in the settings section.

Reviewer: ```suggestion
Once you have restored the database to your environment, go to the backoffice of the project you are migrating to. In the backoffice, you should now see your content in the Content section, Document Types, and Data Types in the Settings section.
```

Taking a closer look at the templates, stylesheets, scripts media, you will notice that it is not there. In the next step we will migrate those over to our new project

> ### step 2: Migrate files

Reviewer: ```suggestion
### Step 2: Migrate Files
```

> In this step, we will migrate our files and media items from our project on the EU region. 

Reviewer: ```suggestion
In this step, we will migrate our files and media items from our project in the EU region. 
```

> 1. Clone down both project to your local machine.

Reviewer: ```suggestion
1. Clone down both the projects to your local machine.
```

2. Run the local East US project and restore the content.

> 3. Open project folders from both **West EU** and **East US**.

Reviewer: ```suggestion
3. Open both the project folders for **West EU** and **East US**.
```

4. Move the view files located in the view folder from **West EU** to the view folder in the **East US** project.

>    - When promted replace the existing files. 

Reviewer: ```suggestion
   - When prompted replace the existing files. 
```
